### ISSUE TITLE
Reset

### ISSUE THREAD

**Issue by VRevol** [nordcloud/cognitocurl#12]
Hi,

I am trying to use this nice tool and I encounter 2 issues (let me know if I should open 2 distinct ones).
For now my environement is Ubuntu 18.04 inside Win10 and seems to be working quite well.

I was able to retrieve a valid token from my Cognito App, so seems promising.

However, our Cognito Pool is created with account status of users set to : FORCE_CHANGE_PASSWORD.
Then for the first login, we must change password.

And it seems this function is not implemented yet:
```bash
$ cognitocurl --cognitoclient <APPID> --userpool <POOL> --token
Username: <user>
Password: *********
TypeError: callback.newPasswordRequired is not a function
    at CognitoUser.authenticateUserInternal (/usr/lib/node_modules/cognitocurl/node_modules/amazon-cognito-identity-js/lib/CognitoUser.js:517:20)
    at /usr/lib/node_modules/cognitocurl/node_modules/amazon-cognito-identity-js/lib/CognitoUser.js:397:21
    at /usr/lib/node_modules/cognitocurl/node_modules/amazon-cognito-identity-js/lib/CognitoUser.js:378:15
    at /usr/lib/node_modules/cognitocurl/node_modules/amazon-cognito-identity-js/lib/Client.js:71:28
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
```

Would it be possible to add this feature ?

Also it would be nice to use this command in a non-interactive way, passing username and password as parameters rather than CLI input.
No sure it is the way you would like this tool to work, but would make my life way easier.

--------------------------------------------------
Second, I tried to use the reset option but I got an error:
```bash
$ cognitocurl --cognitoclient <APPID> --userpool <POOL> --reset
Username: <user>
Password: **************
 /bin/sh: 1: undefined: not found
```

Thanks a lot

**Comment by elgordino** [comment]
I have submitted PR #13 which implements the `newPasswordRequired` functionality.

**Comment by khola** [comment]
Solved by #13 thanks to @elgordino 🍾

**Comment by khola** [comment]
@VRevol regarding the second part (non-interactive input) let's have a separate thread - I will close this one - if you still need the other functionality added please create a new issue and I will work on this next week.
